api/src/main/java/org/openmrs/util/TimeZoneUtil.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.ISODateTimeFormat;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import static org.joda.time.DateTimeZone.UTC;
+
+/**
+ * Helps provide tools to support recommended OpenMRS time zones conventions.
+ * 
+ * @see https://wiki.openmrs.org/display/docs/Time+Zones+Conventions
+ */
+public class TimeZoneUtil {
+	
+	/**
+	 * Formats a date as its RFC 3339 string representation.
+	 * 
+	 * @param date The date.
+	 * @return The date formated as RFC 3339.
+	 */
+	public static String toRFC3339(Date date) {
+		return ISODateTimeFormat.dateTime().print(new DateTime(date.getTime(), UTC));
+	}
+	
+	/**
+	 * Gets the Calendar instance for the date set in UTC. This always returns a GregorianCalendar
+	 * subclass.
+	 * 
+	 * @param date The date.
+	 * @return The GregorianCalendar set in UTC for the date.
+	 */
+	public static Calendar toUTCCalendar(Date date) {
+		return new DateTime(date.getTime(), UTC).toGregorianCalendar();
+	}
+
+
+	/**
+	 * Gets a String date in ISO8601 format. This always returns a Date converted from UTC to the server timezone
+	 * subclass.
+	 *
+	 * @param ISOStringDate A String in a ISO 8601 Format.
+	 * @return Date with the server timezone.
+	 */
+	public static Date toDate(String ISOStringDate) {
```

Comment:
Those utils are always pivoting around a `Date`:
1. Going from a `Date` **to** something, or
2. Going **from** something to a `Date`.

So in this case it's a "from" ISO 8601 `String` to a `Date`. Let us change the signature to:
```java
/**
 * Get a Date out of its ISO 8601 string representation.
 *
 * @param s A date formatted as ISO 8601.
 * @return The Date object.
 */
public static Date fromISO8601(String s) {
```

Surely this should throw some kind of parsing exception, and this should be part of the signature.